src/lib/utilities/create-query-store.ts

```diff
@@ -18,27 +18,26 @@ export const createQueryStore = <T, S = QueryStore<T>>(
       ids: [],
       [key]: {},
     } as unknown as S,
-    createIntervalCallback(update),
-  );
-
-  return store;
-};
+    () => {
```

Comment:
Moving this from a curried utility function to an inline function helps with Svelte's subscription management.